README.md

```diff
@@ -70,15 +70,30 @@ The training data files are hosted in the public "youtube8m-ml" storage bucket
 in the "us-central1" region. Therefore, we've colocated our job in the same
 region in order to have the fastest access to the data.
 
-Generally, you should use a different $JOB_NAME for each of your individual
-runs.
+### Training using GPUs
+
+By default, you will get a single node with CPUs.  To use GPUs, just run:
+
+```sh
+JOB_NAME=yt8m_train_gpu_$(date +%Y%m%d_%H%M%S); gcloud beta ml jobs \
+submit training $JOB_NAME \
+--package-path=youtube-8m --module-name=youtube-8m.train \
+--staging-bucket=gs://$BUCKET_NAME --region=us-central1 \
+--config=youtube-8m/cloudml-gpu.yaml \
```

Comment:
I'd prefer to simplify things by directing people to use a GPU by default and not calling it out specifically. So you can just add the --config flag into the command lines and remove this section. We should also add it into the inference and evaluation commands as there should be a similar speedup factor (but please double check that).